Library/Homebrew/tab.rb

```diff
@@ -242,6 +242,26 @@ def compiler
     super || DevelopmentTools.default_compiler
   end
 
+  def homebrew_tag
+    homebrew_version.sub(/\-\d+\-g([a-f0-9]+)(?:\-dirty)?\Z/, "")
+  end
+
+  def parsed_homebrew_version
+    return Version::NULL if homebrew_version.nil?
+    Version.new(homebrew_tag)
+  end
+
+  # Whether there is reliable runtime dependency information in the receipt.
+  def reliable_runtime_dependencies?
```

Comment:
If we do it now, we'll have to add a line to each of the tests it interferes with that sets the `homebrew_version` to 1.1.6, which should be removed after it's tagged.

That seems even worse than not doing this now to me.